src/test/java/com/connexta/search/index/IndexTests.java

```diff
@@ -187,9 +193,41 @@ void testInvalidAcceptVersion() throws Exception {
   }
 
   @Test
-  @Disabled("TODO")
-  public void testCantReadAttachment() {
-    // TODO verify 400
+  public void testCantReadAttachment() throws IOException {
+    MultipartFile fileMock = mock(MultipartFile.class);
+    doThrow(IOException.class).when(fileMock).getInputStream();
+    doReturn(MediaType.APPLICATION_OCTET_STREAM.toString()).when(fileMock).getContentType();
+    Assertions.assertThrows(
```

Comment:
✏️ `import static org.junit.jupiter.api.Assertions.assertThrows;` for consistency